lib/__tests__/standalone-fix.test.js

```diff
@@ -276,14 +277,15 @@ it('two rules being disabled', async () => {
 		fix: true,
 	});
 
-	const warnings = JSON.parse(result.output)[0].warnings;
+	const warnings = result.results[0].warnings;
 
 	expect(warnings.some((w) => w.text === 'Expected indentation of 0 spaces (indentation)')).toBe(
 		true,
 	);
 	expect(warnings.some((w) => w.text === 'Expected "COLOR" to be "color" (property-case)')).toBe(
 		true,
 	);
+	expect(result.output).toBe('\n		a {\n			COLOR: red;\n		}');
```

Comment:
[suggest] Same here.

```suggestion
	expect(result.output).toBe(code);
```

Reply:
sure, thanks for the suggestion.